tests/fsharpqa/Source/Conformance/InferenceProcedures/NameResolution/RequireQualifiedAccess/OnRecordVsUnion.fs

```diff
@@ -0,0 +1,11 @@
+// #Conformance #TypeInference #Attributes 
+// Verify the RequireQualifiedAccess attribute works on unions
+
+module A =
+    [<RequireQualifiedAccess>]
+    type U = | C
+
+    type C() =
+        static member M() = ()
+
+let x = A.C.M()
```

Comment:
yes pre-PR this is giving an error:

![image](https://cloud.githubusercontent.com/assets/57396/20554982/c15bb2d2-b15f-11e6-8ad7-8f28756c46d4.png)